docs/getting-started.md

```diff
@@ -97,6 +98,46 @@ spec:
     The GitRepository and ScmProvider also need to be installed to the same namespace that you plan on creating PromotionStrategy 
     resources in, and it also needs to be in the same namespace of the secret it references.
 
+## GitLab Configuration
+
+To configure the GitOps Promoter with GitLab, you will need to create a GitLab Access Token with `api` and `write_repository` scopes and configure the necessary resources to allow the promoter to interact with your repository. This Access Token should be used in a secret as follows:
```

Comment:
I had to use the Developer role. That seems to be the lowest role which allows write access to MRs and commit statuses.

```suggestion
To configure the GitOps Promoter with GitLab, you will need to create a GitLab Access Token under the "Developer" role with `api` and `write_repository` scopes and configure the necessary resources to allow the promoter to interact with your repository. This Access Token should be used in a secret as follows:
```